ARSIS-Unity/Assets/Scripts/Components/Elements/NotificationDisplayManager.cs

```diff
@@ -0,0 +1,121 @@
+using ARSIS.EventManager;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.InputSystem;
+using System.Linq;
+using MixedReality.Toolkit.UX;
+using UnityEngine.Rendering.VirtualTexturing;
+using TMPro;
+using UnityEngine.Playables;
+
+public class NotificationDisplayManager : MonoBehaviour, IRenderable
+{
+    private Boolean changed = true;
+    public List<BaseArsisEvent> data = new();
+    [SerializeField]
+    public GameObject MainNotifObj;
+    public GameObject mainParentObject;
+    
+
+
+    private float cooldownTimer = 0f;
+    private float cooldownDuration = 5f; // Cooldown duration in seconds
+
+    public void Render(List<BaseArsisEvent> data)
+    {
+        this.data = data;
+        changed = true;
+    }
+    void Start()
+    {
+        EventDatastore eventDatastore = EventDatastore.Instance;
+        eventDatastore.AddHandler("notification", this);
+    }
+
+    void Update()
+    {
+        cooldownTimer += Time.deltaTime;
+
+        if (cooldownTimer >= cooldownDuration)
+        {
+            cooldownTimer = 0f;
+
+            SetPopUp();
```

Comment:
```suggestion
            //SetPopUp();
```
We have the dedicated window now